src/profession.h

```diff
@@ -18,10 +18,10 @@ class JsonArray;
 class JsonObject;
 class addiction;
 enum add_type : int;
-    class Skill;
-    using skill_id = string_id<Skill>;
+class Skill;
+using skill_id = string_id<Skill>;
 
-    class profession
+class profession
```

Comment:
Uh. So that's where the astyle regression came from. I mean, sure, it's basically an `astyle` bug that it's indenting here, so it's no real fault of yours.

I'll fix that real quick, now that I know what the issue is.